src/components/Navigation/components/Item/Item.tsx

```diff
@@ -287,6 +295,15 @@ export function Item({
           )}
         >
           {itemContentMarkup}
+          {external && (
+            <div className={styles.ExternalIcon}>
+              <Icon
+                accessibilityLabel={externalIconLabel}
+                source={ExternalMinor}
+                color="base"
+              />
+            </div>
+          )}
```

Comment:
Let's move this to a variable so it evaluates before the return.

```suggestion
          {externalLinkIconMarkup}
```